source/routing/index.md

```diff
@@ -30,24 +30,3 @@ or more of the following:
 2. Update a controller so that it represents a particular model.
 3. Change the template on screen, or place a new template into an
    existing outlet.
-
-###Logging Route Changes
```

Comment:
Debugging seems relevant to share here. Maybe something like mentioning the logging option and reminder the reader that the Ember CLI inspector can display routes. A screenshot with the pane open and an arrow to the tab would go a long way.